### PR TITLE
add @npc atcommand

### PR DESCRIPTION
**Pull request by mekolat**
Allows to warp to any npc. Works exactly like `@goto` but for npcs
<hr/>
goes with https://github.com/themanaworld/tmwa-server-data/pull/339

### CODE REVIEW

**Comment by wushin** [comment]
@mekolat wonder if we should Drop the # part of the NPC and make it linus over the same name?

**Comment by mekolat** [comment]
@wushin sure, make a pr on my pr

**Comment by wushin** [comment]
nvm, if someone eventually wants to make a @stallman command in the future it can iterate over NPCs.

File: src/map/atcommand.cpp [hunk 1]
```diff
@@ -701,6 +701,69 @@ ATCE atcommand_goto(Session *s, dumb_ptr<map_session_data> sd,
...
+        }
+        if (sd->bl_m->flag.get(MapFlag::NOWARP)
+            && !(pc_isGM(sd).satisfies(battle_config.any_warp_GM_min_level)))
+        {
+            clif_displaymessage(s,
+                    "You are not authorised to warp you from your actual map."_s);
+            return ATCE::PERM;
+        }
+
+        int x = nd->bl_x, y = nd->bl_y, x0 = (x >= 3)? (x - 3): 0, j = 0,
```
Comment (mekolat) [inline src/map/atcommand.cpp]: should the radius be bigger than 3?

Comment (wushin) [inline src/map/atcommand.cpp]: Well we don't put collision on most our NPCs but I know some, like Hetchel, have collision. anything else I don't think is supposed to be reachable. such as mob spawns or XmasSpawnManager.

Comment (mekolat) [inline src/map/atcommand.cpp]: I know of a banker or two that is behind a collisioned counter

Comment (wushin) [inline src/map/atcommand.cpp]: Wouldn't let me warp to them.

Comment (wushin) [inline src/map/atcommand.cpp]: @mekolat anything larger than 6 is most likely going to be covered so I'd say 5 or 6 max.

Comment (mekolat) [inline src/map/atcommand.cpp]: done

**Pull request merged**